src/Classes/Utils/Utils.ts

```diff
@@ -27,6 +27,49 @@ export class Utils {
         } : null;
     }
 
+    public static hslToRgb(hsl: IHsl) {
+        const result: IRgb = {b: 0, g: 0, r: 0};
+
+        if (hsl.s == 0) {
+            result.b = hsl.l; // achromatic
+            result.g = hsl.l;
+            result.r = hsl.l;
+        } else {
+            const q = hsl.l < 0.5 ? hsl.l * (1 + hsl.s) : hsl.l + hsl.s - hsl.l * hsl.s;
+            const p = 2 * hsl.l - q;
+
+            result.r = Utils.hue2rgb(p, q, hsl.h + 1 / 3);
+            result.g = Utils.hue2rgb(p, q, hsl.h);
+            result.b = Utils.hue2rgb(p, q, hsl.h - 1 / 3);
+        }
+
+        result.r = Math.round(result.r * 255);
+        result.g = Math.round(result.g * 255);
+        result.b = Math.round(result.b * 255);
+
+        return result;
+    }
+
+    private static hue2rgb(p: number, q: number, t: number) {
+        if (t < 0) t += 1;
+        if (t > 1) t -= 1;
+        if (t < 1 / 6) return p + (q - p) * 6 * t;
+        if (t < 1 / 2) return q;
+        if (t < 2 / 3) return p + (q - p) * (2 / 3 - t) * 6;
+        return p;
+    }
+
+    /**
+     * Generate a random RGBA color
+     */
+    public static getRandomColorRGBA(min?: number): IRgb {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public static method not allowed after declaration of private static method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)